### ISSUE TITLE
BungeeCord global-playerlist kicked with "Chat Message Validation Failure"

### ISSUE THREAD

**Issue by MrMango404** [NEZNAMY/TAB#680]
### Server version

BungeeCord

### TAB version

3.1.4

### Describe the bug

When global-playerlist:.enabled: true, player will get kicked with message "Chat Message Validation Failure".
Basically same as https://github.com/NEZNAMY/TAB/issues/673.

### Expected behavior

One player type in chat others get kicked.

### Steps to reproduce

*Atleast two player join the server A & B

1. Installing TAB v3.1.4.jar in bungee
2. Set global-playerlist: . enabled: true
3. A type in chat, B got kicked.

### Checklist

- [X] I am running latest version of the plugin
- [X] I have read the wiki to make sure it's not an issue with configuration
- [X] I have read the Compatibility wiki page and am not trying to run the plugin on an unsupported server version / platform

**Comment by Deichor** [comment]
I also have the same error.

**Comment by MrGraycat** [comment]
This is a version with the commit that adds an option which prevents this from happening:
[https://github.com/NEZNAMY/TAB/actions/runs/2860322153](https://github.com/NEZNAMY/TAB/actions/runs/2860322153)

**Comment by KvasirQc** [comment]
Me too, https://pullanswer.com/questions/getting-kicked-for-chat-message-validation-failiure-on-1-19-1

Regarding to that link, it would be a compatibility problem with essentials.

Beause I run the plugin no chat report on both servers (https://www.spigotmc.org/resources/no-chat-reports-spigot-1-19.102931/)

I am able to run TAB on the server that is not running essentials, but I cannot run tab without that bug happening on the server that is running both TAB and Esssentials :)

I hope I helped you a bit. 

Essentials & paper has said to fixed the bug on their part it seems to be a bug in the TAB plugin at this point.

**Comment by NEZNAMY** [comment]
If you believe you found a bug, open a bug report. Using other issues is chaotic and the issue will be closed when update with fix is released, ignoring any potential sub-reports.

**Comment by SwartZCoding** [comment]
> If you believe you found a bug, open a bug report. Using other issues is chaotic and the issue will be closed when update that fixes original bug is released, ignoring any potential sub-reports.

Same issue here. Verified ! We can't use TAB on 1.19.1 too.

**Comment by NEZNAMY** [comment]
> > If you believe you found a bug, open a bug report. Using other issues is chaotic and the issue will be closed when update that fixes original bug is released, ignoring any potential sub-reports.
> 
> Same issue here. Verified ! We can't use TAB on 1.19.1 too.

You were using bungeecord, weren't you? Are you reporting the original issue?

**Comment by SwartZCoding** [comment]
> > > If you believe you found a bug, open a bug report. Using other issues is chaotic and the issue will be closed when update that fixes original bug is released, ignoring any potential sub-reports.
> > 
> > 
> > Same issue here. Verified ! We can't use TAB on 1.19.1 too.
> 
> You were using bungeecord, weren't you? Are you reporting the original issue?

I just confirm the issue of [MrMango404](https://github.com/MrMango404) yes.

On the other hand, it does not change the fact that originally if we use tab, the error occurs and if we delete it we are no longer kicked.

To fix this, check this open-source project : https://www.spigotmc.org/resources/%E2%9C%A8-antipopup-no-chat-reports-and-popup-%E2%9C%A8.103782/.

Personally i just play with packet to disable mojang chat verification and this is fix my issue if i reinstall Tab.

**Comment by NEZNAMY** [comment]
Oh that I fixed already, I thought you are reacting to a comment which apparently got deleted saying it happens on paper.

**Comment by RedZephon** [comment]
> Oh that I fixed already, I thought you are reacting to a comment which apparently got deleted saying it happens on paper. My comment about chaos is even more on point now.

I don't believe that this issue is fixed. I along with many others are still facing this issue. If you search for this issue on Reddit you can see lots of people still having this issue and uninstalling TAB to get it fixed.

For me, this problem is solved by having players log in with 1.19.0 but anyone using 1.19.2 gets kicked when any chat message is sent.

**Comment by NEZNAMY** [comment]
Yes, as you can see, this issue has 3.1.5 milestone which will have this fixed.

**Comment by IceWaffles** [comment]
So this shouldn't be affecting paper whatsoever?

**Comment by NEZNAMY** [comment]
Correct.
